recipes/pudb/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set version = "2016.2" %}
 
-ackage:
+package:
```

Comment:
By smarter, maybe we should spell check or at least say unsupported key. Thoughts @pelson?